.github/dependabot.yml

```diff
@@ -6,35 +6,14 @@ updates:
       interval: "daily"
     allow:
       - dependency-type: "all"
```

Comment:
What do you think about limiting this only to direct dependencies:
```yaml
dependency-type: "direct"
```

And then we can add another `updates` block for security updates, which would also update indirect dependencies:
```yaml
- package-ecosystem: "gomod"
    directory: "/"
    schedule:
      interval: "daily"
    allow:
      - dependency-type: "all"
    open-pull-requests-limit: 0
```